src/DatumConverter.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+namespace r;
+
+use ReflectionFunction;
+use r\Query;
+use r\Datum\ArrayDatum;
+use r\Datum\BoolDatum;
+use r\Datum\NullDatum;
+use r\Datum\NumberDatum;
+use r\Datum\ObjectDatum;
+use r\Datum\StringDatum;
+use r\Exceptions\RqlDriverError;
+use r\FunctionQuery\RFunction;
+use r\Queries\Dates\Iso8601;
+use r\ValuedQuery\MakeArray;
+use r\ValuedQuery\MakeObject;
+use r\ValuedQuery\RVar;
+
+class DatumConverter
```

Comment:
While you're technically correct, it means changing the whole class (as there are many interclass calls). I see we're calling new in all of 3 places. Perhaps we can revisit this in the future?
```
# grep -rn new\ DatumConverter src/
src/Cursor.php:143:        $dc = new DatumConverter;
src/rdb/global.php:162:    $dc = new DatumConverter;
src/rdb/global.php:174:    $dc = new DatumConverter;
```